launcher/proto/doc.go

```diff
@@ -0,0 +1,21 @@
+// Package proto contains protocol buffers for the attestation
+//
+// Generating Protocol Buffer Code
+//
+// Anytime the Protocol Buffer definitions change, the generated Go code must be
+// regenerated. This can be done with "go generate". Just run:
+//   go generate ./...
+//
+// Upstream documentation:
+// https://developers.google.com/protocol-buffers/docs/reference/go-generated
+//
+// Code Generation Dependencies
```

Comment:
We should also mention:
  - That the google/api code has to be copied in from some upstream
  - That `protoc-gen-go-grpc` also has to be installed via `go install google.golang.org/grpc/cmd/protoc-gen-go-grpc`

Reply:
good point, done